lib/config/rules/stylistic-issues.js

```diff
@@ -129,6 +135,8 @@ module.exports = {
   'operator-linebreak': ['warn', 'after', {overrides: {'?': 'before', ':': 'before'}}],
   // Enforce padding within blocks
   'padded-blocks': 'off',
+  // require or disallow padding lines between statements <FEEDBACK>
+  'padding-line-between-statements': 'off',
```

Comment:
This rule has lots of config options: https://eslint.org/docs/rules/padding-line-between-statements
I'd say just keep it off.